app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -195,6 +196,7 @@ abstract class AppDatabase : RoomDatabase() {
         purgeDeleted()
         purgeDeletedPhoneNumbers()
         purgeDeletedBusinessIds()
+        purgeDeletedPatientAfterRetentionTime(Instant.now())
```

Comment:
Let's add a test to verify this purge. Please, refer to `PurgeDatabaseAndroidTest`

Reply:
Can we also verify that the associated records (Blood pressures and sugars, Appointments, Prescribed drugs, Medical history) are also deleted when a patient record fetched from online lookup is deleted?

Reply:
Yes, realised this when reviewing a different PR. Will update to deleting the records attached to it as well.